scripts/pre_commit_linter.py

```diff
@@ -535,6 +535,9 @@
     '/scripts/install_third_party_libs.py',
     '/.github/']
 
+# Check if project is running in CircleCI.
+IS_CI = os.getenv('CIRCLECI')
```

Comment:
Also I don't think this is the correct approach. The script should not need to know about its context.

Instead, in the instructions for circleci, call this script with the relevant args such that only the python linting is run (you might need to modify this script a bit in order to support this).

Also I largely don't understand what's going on in the `IS_CI = True` case. In particular, why are we ignoring all the other checks (JS, TS, CSS, etc.) in that case, and where are those checks going to happen instead?

Reply:
When `IS_CI = True` all other checks (JS, TS, CSS, etc.) will run in Code Climate.